.github/workflows/graphql_sdk_generate.yml

```diff
@@ -2,7 +2,7 @@ name: Generate Graphql JS SDK
 
 on:
   push:
-    branches: [main]
+    branches: [chore]
```

Comment:
revert this?

Reply:
doing test my dev branch now . will revert this later.